README.md

```diff
@@ -126,10 +126,10 @@ Note that the Spectrum CSS UI icons must change as well, see below for a full ex
 
 ### Importing Icons
 
-To get icons, you'll need to use the `AS.loadIcons()` function (also available as `AdobeSpectrum.loadIcons()`). This function lives in the `icons/AS.loadIcons.js` file and has the following signature:
+To get icons, you'll need to use the `loadIcons()` function. This function lives in the `icons/AS.loadIcons.js` file and has the following signature:
```

Comment:
It's now `icons/loadIcons.js`

Reply:
But maybe we should just tell them to use https://www.npmjs.com/package/loadicons ?

Reply:
Hmm.... yes maybe... so we should turf it from the repo then?

Reply:
It's included in the `dist/`, but the file is actually pulled from an npm install of `loadicons`, so we're good there. I think this is fine for now!